Resources/Prototypes/Entities/Markers/Spawners/Random/toy.yml

```diff
@@ -30,6 +30,7 @@
         - PlushieSharkGrey
         - ToyAmongPequeno
         - ToyMouse
+        - Mobayadoll
```

Comment:
Should use [PascalCase](https://en.wikipedia.org/wiki/Camel_case) naming like so: `MobAyaDoll`.
All other instances of `Mobayadoll` would have to be changed to fit this.

Reply:
Also should not be equal chance to other toys